src/Info.plist.in

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This whole file looks like some sort of input to something to make an `Info.plist` file - I'm not a MacOS expert so how/where is that done?